src/data/docs/documentation.md

```diff
@@ -5,257 +5,232 @@ description: How to write Unison documentation and associate it with your code
+      None {{ Fruit flies like a banana. }} 
+    }}
+    
+    A callout with icon: 
+    {{ docCallout
+      (Some {{ 🌹 }})
+      {{
+      Rose is a rose is a rose is a rose.
+      }} 
+    }}
+  }} 
+```
+
+Asides are for extra details that don't belong in the main text, whereas callouts should be used where you'd like to emphasize a particular point. Callouts are parameterized by an `Optional` icon so you can further customise your messages. 
+
+Block quotes, tables and hoverable tool-tips are also supported. 
 ```unison
-List.take.doc = [:
-  `@List.take n xs` returns a list of the first `n`
-  elements of `xs`. Complexity is O(log n).
-
-  Here's a typical example:
-
-      @[source] List.take.examples.ex1
-      ↳ @[evaluate] List.take.examples.ex1
-
-  If we `@List.take` more than the size of the list, we
-  get back the original list:
-
-      @[source] List.take.examples.ex2
-      ↳ @[evaluate] List.take.examples.ex2
-
-  And lastly, `@List.take 0 xs` is always equal to `[]`,
-  the empty list.
-
-  __Also see:__
-
-    * @List.drop (ignores the first `n` elements)
-    * @List.doc has more about the @List data structure
-:]
-```
-
-That is:
-
-- Start with a brief overview, then maybe a longer description, then some examples, then link to related definitions and further reading.
-- Use Markdown syntax for formatting. Indent examples by 4 spaces for readability when viewing the docs in plain text form, and this will also render nicely as Markdown.
-- Follow sensible naming conventions for associated documentation and examples. For a definition `List.take`, for example:
-    - Its primary documentation value could be called `List.take.doc`, and secondary docs could be in the `List.take.docs` namespace. Perhaps a document called `List.take.docs.advancedUsages` could show advanced usages of the function.
-    - Examples could be in the `List.take.examples` namespace. For instance, the examples above were named `List.take.examples.ex1` and `List.take.examples.ex2`.
-
+moreDocFeatures = {{
+  {{ docBlockquote
+    {{
+      "And what is the use of a book," thought Alice, "without
+      pictures or conversation?"
+      
+      *Lewis Carroll, Alice's Adventures in Wonderland*
+    }} 
+  }}
+    
+  {{ docTooltip {{ Knock Knock! }} {{ Who's there? }} }}
+    
+  {{ docTable [
+    [ {{ Twas brillig, }}, {{ and the slithy toves }}, {{
+      Did gyre and gimble in the wabe: All mimsy were the borogoves,
+    }} ],
+    [ {{ And the }}, {{ mome raths }}, {{ outgrabe. }} ] 
+  ] }}
+}}
+```
+
+We hope you'll agree that writing documentation in Unison is an easy and delightful process that will help developers better communicate the intent and value of their work. More details on documentation can be found [here.](https://github.com/unisonweb/unison/blob/ca951f36dbdc8a32e267acb9f8051ef64b90ec97/unison-src/transcripts-using-base/doc.output.md) We look forward to reading more Unison docs in the libraries and code of your own making. 😃 
```

Comment:
Point of usability: links that just consist of the word "here" are hard to click and it's not obvious what they go to. Better: `[The document format specification](https://github.com/unisonweb/unison/blob/ca951f36dbdc8a32e267acb9f8051ef64b90ec97/unison-src/transcripts-using-base/doc.output.md) has more details`.